src/pages/library/[id].tsx

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/jsx-indent-props */
+/* eslint-disable react/jsx-indent */
+import { useState } from "react";
+import { IoMdArrowBack } from "react-icons/io";
+
+import {
+    Avatar,
+    Box,
+    Button,
+    Flex,
+    IconButton,
+    Stack,
+    Tag,
+    Text,
+} from "@chakra-ui/react";
+import { doc, getDoc } from "firebase/firestore";
+import { GetServerSideProps } from "next";
+import { useSession } from "next-auth/react";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { ParsedUrlQuery } from "querystring";
+
+import { useBooks } from "../../hooks/useBooks";
+import { db } from "../../services/firebase";
+import { borrowBook } from "../../store/books/actions";
+
+export default function Library({ book }) {
+    const { data: session } = useSession();
+    const router = useRouter();
+    const { isLoading } = useBooks();
+    const handleBorrowBook = async () => {
+        await borrowBook(book, session.user);
+        router.reload();
```

Comment:
o motivo do reload é pra carregar o quê? talvez seja interessante dar os refresh's necessários sem recarregar a página (é esquisito ver a página recarregar literalmente)

Reply:
Quando pego emprestado o livro, as informações sobre o livro não alteram. Ai eu forço renderização. Obviamente não é a melhor prática, mas enquanto não deixei como estado, não é possível.

Reply:
Mas vou fazer isso numa feature futura.